src/components/GameAbout/index.tsx

```diff
@@ -26,6 +32,13 @@ const GameAbout = ({ description, classnames }: GameAboutProps) => {
 
   return (
     <div className={classnames?.container}>
+      {gameName && (
+        <div className={styles.gameTitle}>
+          <h2 className={classNames('h4', styles.title, classnames?.gameTitle)}>
```

Comment:
could we just make this an h4 element instead of applying an h4 class to an h2 element?